src/utils/flattenMetadataQueryResponse.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @flow
+ * @file File for some simple dom utilities
```

Comment:
Can the filename be more generic, similar to `util/file.js`, in case we need additional utils in the future? Is this docstring outdated?

Reply:
This file is specific to flatten out the metadata query response.  Like `utils/flatten.js`